doc/manual/embedding.rst

```diff
@@ -29,8 +29,9 @@ We start with a simple C program that initializes Julia and calls some Julia cod
            julia time to cleanup pending write requests
            and run all finalizers
       */
-      jl_atexit_hook();
-      return 0;
+      int ret = 0;
+      jl_atexit_hook(ret);
```

Comment:
At least rename ``ret`` to ``exitcode``, but it really seems unnecessary. You don't end an example ``main()`` function in C with:
```
int exitcode = 0;
return exitcode;
```